leases/lease.go

```diff
@@ -0,0 +1,50 @@
+package leases
+
+import (
+	"encoding/json"
+	"time"
+)
+
+const (
+	TimeFormat = time.RFC3339
```

Comment:
We actually do have a standard in [deis/pkg](https://github.com/deis/pkg/blob/master/time/time.go#L6).

Since this project it's a closed system, I don't think we need to use it, and I certainly don't want to pull in that dependency just to get access to a single constant.